templates/goto.tmpl.php

```diff
@@ -334,7 +334,7 @@
 			<?php if (User::isLoggedIn()) : ?>
 				<div class="notification">
 					<input type="checkbox" id="check_notif" class="checkbox" <?php if ($notification !== null) print_unescaped(' checked'); ?> />
-					<label for="check_notif"><?php p($l->t('Receive notification email on activity')); ?></label>
```

Comment:
We should keep this short for preventing line break in the mobile view.

Reply:
What about `Receive notifications per email`?

Reply:
Added another commit, and simplified this. :)